### PR TITLE
Added local_hpc compatibility with Zimo's update

### PR DESCRIPTION
**Pull request by mitkotak**

### CODE REVIEW

File: cybergis_compute_client/UI.py [hunk 1]
```diff
@@ -564,7 +564,9 @@ def on_click(change):
 
             self.compute.login()
             localDataFolder = None
-            self.jupyter_globus = self.compute.get_user_jupyter_globus()
+            data = self.get_data()
+            if data['computing_resource'] != 'local_hpc':
```
Comment (alexandermichels) [inline cybergis_compute_client/UI.py:599]: So if I'm understanding this PR correctly, this is the major change and the rest are just flake8 stuff?

Comment (mitkotak) [reply cybergis_compute_client/UI.py:599]: Pretty much. I had hoped that the flake8 stuff would be resolved by the time this PR would be merged but I am happy to create a separate PR for that if you want.

Comment (alexandermichels) [reply cybergis_compute_client/UI.py:599]: Okay, just making sure I understand what's going on! I'll test it out.

**Comment by alexandermichels** [comment]
@mitkotak I just merged Taylor's commit which fixed flake8 errors and brought the latest v2 changes to this branch. Could you resolve the conflicts that caused with both of your PRs?

**Review by alexandermichels — APPROVED** [review]
Code makes sense and everything seems to run on dev. Great work @mitkotak!

**Pull request merged**